### PR TITLE
UI Updates

### PR DESCRIPTION
**Pull request by jlaureano**

### CODE REVIEW

**Comment by eliashdezr** [comment]
Is there a summary of these changes? :)

**Comment by topcatx** [comment]
screenshots with UI updates would be nice..

**Comment by jlaureano** [comment]
screen shot 2018-01-05 at 7 41 37 pm
screen shot 2018-01-05 at 7 41 44 pm
screen shot 2018-01-05 at 7 41 57 pm
screen shot 2018-01-05 at 7 42 01 pm
screen shot 2018-01-05 at 7 42 10 pm
screen shot 2018-01-05 at 7 42 22 pm
screen shot 2018-01-05 at 7 42 29 pm
screen shot 2018-01-05 at 7 42 43 pm
screen shot 2018-01-05 at 7 42 53 pm
screen shot 2018-01-05 at 12 03 10 pm

**Comment by Tommixoft** [comment]
Wow compared to windows wallet this one is MASTERPIECE!

**Comment by jlaureano** [comment]
https://medium.com/@jlaureanoalvarez/the-case-for-usability-649cd1e1cc59 @Tommixoft

**Comment by paradoxicalNL** [comment]
Very near nice, clean look!

File: .gitignore [hunk 1]
```diff
@@ -17,6 +17,9 @@ src/config/verge-config.h
 src/VERGEd
 src/qt/VERGE-qt
 
+VERGEd
```
Comment (ekryski) [inline .gitignore]: Yeah good call. On OS X it's easy to accidentally commit these.

**Comment by ekryski** [comment]
I checked this out. Looks great @jlaureano!

@justinvforvendetta totally :shipit:

**Comment by ekryski** [comment]
@jlaureano the only minor change I was going to suggest is not making the "decode transaction" and "jump to block" buttons full width.

**Comment by jlaureano** [comment]
@ekryski can we chat via twitter? @ joselaureano

**Comment by IgorKonovalov** [comment]
Great work! I'm excited to see this in action!

**Comment by jlaureano** [comment]
@ekryski added to the pull request

**Comment by paradoxicalNL** [comment]
Is it easy to push this to Linux distribution?
Gonna check that out!

Op 6 jan. 2018 4:04 p.m. schreef "Jose Laureano" <notifications@github.com>:

**Comment by topcatx** [comment]
@paradoxicalNL Qt being multi-platform, this change will be available on all platforms, each OS using their own native look and feel style. 

👍 Nice work !!! @jlaureano

**Comment by paradoxicalNL** [comment]
Very nice work!! +100

2018-01-07 0:47 GMT+01:00 topcatx <notifications@github.com>:

**Pull request merged**